pyeudiw/satosa/backend.py

```diff
@@ -176,14 +176,17 @@ def pre_request_endpoint(self, context, internal_request, **kwargs):
         # PAR
         payload = {
             'client_id': self.client_id,
-            'request_uri': self.absolute_request_url
+            'request_uri': self.absolute_request_url,
+            'session_id': context.state["SESSION_ID"]
```

Comment:
This is not in the technical specs and neither in the PAR RFC, as already said we MUST not have the session_id in this payload